tests/queries/saved_queries/commands_tests.py

```diff
@@ -108,3 +120,109 @@ def test_export_query_command_key_order(self, mock_g):
             "version",
             "database_uuid",
         ]
+class TestImportSavedQueriesCommand(SupersetTestCase):
+    def test_import_v1_saved_queries(self):
+        """Test that we can import a saved query"""
+        contents = {
+            "metadata.yaml": yaml.safe_dump(saved_queries_metadata_config),
+            "databases/imported_database.yaml": yaml.safe_dump(database_config),
+            "queries/imported_query.yaml": yaml.safe_dump(saved_queries_config)
+        }
+
+        command = ImportSavedQueriesCommand(contents)
+        command.run()
+
+        saved_query = db.session.query(SavedQuery).filter_by(
+            uuid=saved_queries_config["uuid"]
+        ).one()
+        assert saved_query.schema == "public"
+        assert saved_query.sql == (
+            """
+            -- Note: Unless you save your query, 
+            these tabs will NOT persist if you clear
+            your cookies or change browsers.
+
+            SELECT * from birth_names
+            """
+        )
+        assert saved_query.uuid == "05b679b5-8eaf-452c-b874-a7a774cfa4e9"
+        assert saved_query.database_uuid == "b8a1ccd3-779d-4ab7-8ad8-9ab119d7fe89"
```

Comment:
I suspect these will change every time you run the tests, so you might need to remove these tests.